components/Trade/Advanced/TradingPanel/Account/AccountModal.tsx

```diff
@@ -48,13 +48,6 @@ const ModalButton = styled(Button)`
     width: 80%;
     margin: auto;
     height: 40px;
-    color: var(--color-text);
-
-    &:disabled {
-        &:hover {
-            background: none;
-        }
-    }
```

Comment:
Will approve but am trusted this has been tested thoroughly I remember putting it in for some reason but it may have changed